src/database-controller/sdk/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "openpaidbsdk",
+  "version": "1.0.0",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "eslint --ext .js --ext .jsx .",
+    "lintfix": "eslint --ext .js --ext .jsx . --fix"
```

Comment:
it's duplicate, you could use `npm run lint -- --fix`

Reply:
Done.